documentation/sphinx/source/perpetual-storage-wiggle.rst

```diff
@@ -29,21 +29,25 @@ With a perpetual wiggle, storage migrations will be much less impactful. The wig
 Configuration
 =============
 
-You can configure the Perpetual Storage Wiggle via the FDB :ref:`command line interface <command-line-interface>`.
+You can configure the Perpetual Storage Wiggle via ``fdbcli`` :ref:`command line interface <command-line-interface>`.
 
-Note that to have the Perpetual Storage Wiggle change the storage engine type, you must configure ``storage_migration_type=gradual``.
+To enable the Perpetual Storage Wiggle feature, you must configure ``storage_migration_type=gradual perpetual_storage_wiggle=1``.
```

Comment:
perpetual_storage_wiggle=1 
The above command can enable the perpetual storage wiggle feature.

`storage_migration_type=gradual perpetual_storage_wiggle=1``.
If you want the Perpetual Storage Wiggle change the storage engine type, you must configure also add command ``storage_migration_type=gradual``